content/references/protocol-reference/transactions/transaction-types/nftokenacceptoffer.md

```diff
@@ -3,7 +3,7 @@ html: nftokenacceptoffer.html
 parent: transaction-types.html
 blurb: Accept an offer to buy or sell an NFToken.
 labels:
-  - NFTs, Non-fungible Tokens
+  - NFTokens, Non-fungible Tokens
```

Comment:
If you change this label it won't work as a link anymore, so I suggest reverting this instance